guide/ops/catalog/index.md

```diff
@@ -160,14 +179,15 @@ and its implementation will be the Java class `org.apache.brooklyn.entity.nosql.
 ~~~ yaml
 brooklyn.catalog:
   id: datastore
-  version: 1.0
+  version: "1.0"
   itemType: template
   iconUrl: classpath://org/apache/brooklyn/entity/nosql/riak/riak.png
   name: Datastore (Riak)
   description: Riak is an open-source NoSQL key-value data store.
   item:
-    type: org.apache.brooklyn.entity.nosql.riak.RiakNode
-    name: Riak Node
+    services:
```

Comment:
It's not made clear why templates use a `services` block but entities do not. Is this covered somewhere else?